repos/fdbt-site/src/pages/selectCaps.tsx

```diff
@@ -0,0 +1,167 @@
+                                    </div>
+                                </div>
+
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className="govuk-radios__input"
+                                        id="valid-days-not-required"
+                                        name="capChoice"
+                                        type="radio"
+                                        value="no"
+                                        data-aria-controls="conditional-caps-2"
+                                        defaultChecked={!selectedId}
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="no-choice">
+                                        No
+                                    </label>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </div>
+
+                    <input
+                        type="submit"
+                        value="Continue"
+                        id="continue-button"
+                        className="govuk-button govuk-!-margin-right-2"
+                    />
+
+                    <a className="govuk-button govuk-button--secondary" href="/viewCaps">
+                        Create new
+                    </a>
+                </>
+            </CsrfForm>
+        </FullColumnLayout>
+    );
+};
+
+const CapsCard = ({ caps, selectedId }: { caps: CapInfo; selectedId: number | null }): ReactElement => {
+    return (
+        <div className="card">
+            <div className="card__body caps">
+                <div className="govuk-radios">
+                    <div className="govuk-radios__item card__selector">
+                        <input
+                            className="govuk-radios__input"
+                            id={`${caps.cap.name}-radio`}
+                            name="cap"
+                            type="radio"
+                            value={caps.id}
+                            aria-label={caps.cap.name}
+                            defaultChecked={selectedId === caps.id}
+                        />
+                        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                        <label className="govuk-label govuk-radios__label" />
+                    </div>
+                </div>
+                <CapCardBody entity={caps} />
+            </div>
+        </div>
+    );
+};
+
+export const getServerSideProps = async (ctx: NextPageContextWithSession): Promise<{ props: SelectCapsProps }> => {
+    const csrfToken = getCsrfToken(ctx);
+
+    let errors: ErrorInfo[] = [];
+    const capsDefinition = getSessionAttribute(ctx.req, CAPS_DEFINITION_ATTRIBUTE);
+
+    if (capsDefinition && 'errors' in capsDefinition) {
+        errors = capsDefinition.errors;
+    }
```

Comment:
const capsDefinition = getSessionAttribute(ctx.req, CAPS_DEFINITION_ATTRIBUTE);
const errors = !!capsDefinition && 'errors' in capsDefinition ? capsDefinition.errors : [];